samples/CMakeLists.txt

```diff
@@ -86,26 +86,26 @@ if(   (${CMAKE_CXX_COMPILER_ID} MATCHES "GNU")
   set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O2")
 
   # Handles x86 compilation support on x64 arch.
-  if(${PLATFORM} MATCHES 32)
+  if(${PLATFORM} EQUAL 32)
     set(CMAKE_CXX_FLAGS        "${CMAKE_CXX_FLAGS} -m32")
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -m32")
   endif()
-elseif(${CMAKE_CXX_COMPILER_ID} MATCHES "MSVC")
-  set(CMAKE_CXX_FLAGS         "${CMAKE_CXX_FLAGS} /Za")
-  set(CMAKE_CXX_FLAGS_DEBUG   "${CMAKE_CXX_FLAGS_DEBUG} /w2 /DEBUG")
+elseif(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
+  set(CMAKE_CXX_FLAGS         "${CMAKE_CXX_FLAGS}")
```

Comment:
The 94th ? yes it is, but maybe it is more elegant to keep it for symmetry or if one wants to add a flag with ease.